### PR TITLE
Prepare 3.x major release

### PR DESCRIPTION
**Pull request by core23**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -2,6 +2,17 @@
 All notable changes to this project will be documented in this file.
 This project adheres to [Semantic Versioning](http://semver.org/).
 
+## [3.0.0-alpha-1](https://github.com/sonata-project/SonataSeoBundle/compare/2.14.0...3.0.0-alpha-1) - 2021-10-11
```
Comment (core23) [inline CHANGELOG.md]: Not sure if the compare url is correct. Should we use the last tagged 2.x release?

Comment (VincentLanglet) [reply CHANGELOG.md]: Seems good, https://github.com/sonata-project/SonataAdminBundle/blob/4.x/CHANGELOG.md#400-alpha-1---2021-04-11

**Review by VincentLanglet — DISMISSED** [review]

File: CHANGELOG.md (hunk 1 above)
Comment (core23) [reply CHANGELOG.md]: Is `3.0.0-alpha-1` a valid version? Did not find any symfony alpha releases. There are only some `*-BETA1` versions.

Comment (VincentLanglet) [reply CHANGELOG.md]: 3.0.0-alpha.1 is better

**Review by VincentLanglet — APPROVED** [review]

**Review by franmomu — APPROVED** [review]

**Pull request merged**